<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="172.16.0.2"/>
  <arg name="load_gripper" default="false" />

  <arg name="robot" default="panda" doc="choose your robot. Possible values: [panda, fr3]"/>
  <arg name="arm_id" default="$(arg robot)" />

  <include file="$(find franka_control)/launch/franka_control.launch" pass_all_args="true"/>
  <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="cartesian_position_force_hybird_controller"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_example_controllers)/launch/rviz/franka_description_with_marker.rviz -f $(arg arm_id)_link0 --splash-screen $(find franka_visualization)/splash.png"/>

  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" required="false" />

  <!-- <include file="$(find azure_kinect_ros_driver)/launch/driver.launch" ></include> -->
   <node pkg="robotiq_ft_sensor" type="rq_sensor" name="rq_sensor"/>
  <!-- <include file="$(find orbbec_camera)/launch/gemini2.launch" ></include> -->
</launch>
